Suko/Model/SUKAnime.h

```diff
@@ -9,24 +9,38 @@
 #import <Parse/Parse.h>
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Model used to represent an anime.
+ */
 @interface SUKAnime : NSObject
 
 /** Corresponding MyAnimeList ID */
 @property (nonatomic) int malID;
```

Comment:
To prevent your properties from potentially getting altered by other classes, you can add the `readonly` attribute to the property (i.e. `@property (nonatomic, readonly) int malID`).